packages/nodes-base/nodes/SendInBlue/AttributeDescription.ts

```diff
@@ -0,0 +1,531 @@
+			},
+		],
+	},
+];
+
+const updateAttributeOperations: INodeProperties[] = [
+	{
+		displayName: 'Category',
+		name: 'updateAttributeCategory',
+		default: 'calculated',
+		description: 'Category of the attribute',
+		displayOptions: {
+			show: {
+				resource: ['attribute'],
+				operation: ['updateAttribute'],
+			},
+		},
+		options: [
+			{
+				name: 'Calculated',
+				value: 'calculated',
+			},
+			{
+				name: 'Category',
+				value: 'category',
+			},
+			{
+				name: 'Global',
+				value: 'global',
+			},
+		],
+		type: 'options',
+	},
+	{
+		displayName: 'Name',
+		name: 'updateAttributeName',
+		default: '',
+		description: 'Name of the existing attribute',
+		displayOptions: {
+			show: {
+				resource: ['attribute'],
+				operation: ['updateAttribute'],
+			},
+		},
+		type: 'string',
+	},
+	{
+		displayName: 'Value',
+		name: 'updateAttributeValue',
+		default: '',
+		description: 'Value of the attribute to update',
+		displayOptions: {
+			show: {
+				resource: ['attribute'],
+				operation: ['updateAttribute'],
+			},
+			hide: {
+				updateAttributeCategory: ['category'],
+			},
+		},
+		type: 'string',
+		placeholder: '',
+		routing: {
+			send: {
+				type: 'body',
+				property: 'value',
+				value: '={{$value}}',
+			},
+		},
+	},
+	{
+		displayName: 'Update Fields',
+		name: 'updateAttributeCategoryList',
+		default: {},
+		description: 'List of the values and labels that the attribute can take',
+		type: 'collection',
+		placeholder: 'Add Field',
+		displayOptions: {
+			show: {
+				resource: ['attribute'],
+				operation: ['updateAttribute'],
+				updateAttributeCategory: ['category'],
+			},
+		},
+		options: [
+			{
+				displayName: 'Contact Attributes',
+				name: 'updateCategoryEnumeration',
+				placeholder: 'Add Attribute',
+				type: 'fixedCollection',
+				typeOptions: {
+					multipleValues: true,
+				},
+				options: [
+					{
+						name: 'updateAttributesValues',
+						displayName: 'Attribute',
+						values: [
+							{
+								displayName: 'Value',
+								name: 'attributeCategoryValue',
+								type: 'number',
+								default: 1,
+								description: 'ID of the value, must be numeric',
+								routing: {
+									send: {
+										value: '={{$value}}',
+										property: '=enumeration[{{$index}}].value',
+										type: 'body',
+									},
+								},
+							},
+							{
+								displayName: 'Label',
+								name: 'attributeCategoryLabel',
+								type: 'string',
+								default: '',
+								routing: {
+									send: {
+										value: '={{$value}}',
+										property: '=enumeration[{{$index}}].label',
+										type: 'body',
+									},
+								},
+								description: 'Label of the value',
+							},
+						],
+					},
+				],
+				default: {},
+				description: 'List of values and labels that the attribute can take',
+			},
+		],
+	},
+];
+
+const deleteAttribueOperations: INodeProperties[] = [
+	{
+		displayName: 'Category',
```

Comment:
Is the category necessary when removing an attribute?

Reply:
Yes it's required, I updated the field to be `{required:true}`